### DISCUSSION TITLE
Nginx Proxy Manager using DNS01 challenge - missing certbot plugin

### DISCUSSION THREAD

**Discussion by githubsean** [tteck/Proxmox#2902] [General] [answered]
After installing NPM I found I couldn't setup a Letsencrypt certificate using DNS authentication and CloudFlare. The error returned by certbot was fairly clear - missing CloudFlare plugin.

From the LXCs console I simply installed the relevant package: python3-certbot-dns-cloudflare

I did an advanced install, chose Debian Bullseye. I'm not sure if the certbot package defaults to including it in the other distributions.

Anyway, once I got over that hurdle, it all worked well.

**Comment by tteck** [comment, marked as answer]
https://github.com/tteck/Proxmox/commit/bb965d681caae2962573e008d4f5a5682a2079ff
